<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatistikDohod</class>
 <widget class="QWidget" name="StatistikDohod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DohodStatistic</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sceneRect">
    <rectf>
     <x>0.000000000000000</x>
     <y>0.000000000000000</y>
     <width>619.000000000000000</width>
     <height>282.000000000000000</height>
    </rectf>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>  Январь | Февраль |   Март  |  Апрель |    Май   |   Июнь  |   Июль  | Август | Сентябрь | Октябрь | Ноябрь | Декабрь</string>
   </property>
  </widget>
  <widget class="QSlider" name="ScaleSlider">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Масштаб</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>121</width>
     <height>161</height>
    </rect>
   </property>
   <property name="sceneRect">
    <rectf>
     <x>0.000000000000000</x>
     <y>0.000000000000000</y>
     <width>79.000000000000000</width>
     <height>159.000000000000000</height>
    </rectf>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateStart">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::YearSection</enum>
   </property>
   <property name="displayFormat">
    <string>yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>От</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>До</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEnd">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::YearSection</enum>
   </property>
   <property name="displayFormat">
    <string>yyyy</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
